django/contrib/admin/templates/admin/app_list.html

```diff
@@ -1,44 +1,46 @@
 {% load i18n %}
 
-{% if app_list %}
-  {% for app in app_list %}
-    <div class="app-{{ app.app_label }} module{% if app.app_url in request.path|urlencode %} current-app{% endif %}">
-      <table>
-        <caption>
-          <a href="{{ app.app_url }}" class="section" title="{% blocktranslate with name=app.name %}Models in the {{ name }} application{% endblocktranslate %}">{{ app.name }}</a>
-        </caption>
-        {% for model in app.models %}
-          {% with model_name=model.object_name|lower %}
-            <tr class="model-{{ model_name }}{% if model.admin_url in request.path|urlencode %} current-model{% endif %}">
-              <th scope="row" id="{{ app.app_label }}-{{ model_name }}">
-                {% if model.admin_url %}
-                  <a href="{{ model.admin_url }}"{% if model.admin_url in request.path|urlencode %} aria-current="page"{% endif %}>{{ model.name }}</a>
+<div id="content-main" class="filterable-apps-table">
```

Comment:
I'm not sure this is correct. When this template is included in nav_sidebar.html, this div with `id="content-main"` gets included inside the `<nav>` element and that does not feel correct, more so knowing that there is another div with the same id further down the page.

I would suggest to completely revert changes to this file, and only add the div in the index.html file where app_list.html is included.